js/main.js

```diff
@@ -1,2 +1,167 @@
 $(function () {
+  // Constants and Variables
+  const rockPaperScissorsChoices = [
+    "Rock",
+    "Paper",
+    "Scissors"
+  ]
+
+  let result
+  let playerChoice
+  let botChoice
+  let resultMessage
+  let playerScore = 0
+  let botScore = 0
+  let playerRound = 0
+  let botRound = 0
+  let playerBonus = 0
+  let botBonus = 0
+  let bonusFlagBot = false
+  let bonusFlagPlayer = false
+  let winLog = []
+
+  // Buttons
+  let $choices = $("div.buttons button")
+
+  // When a button is clicked
+  $choices.on('click', function(){
+
+    // Determine choices
+    playerChoice = $(this).text()
+    botChoice = rockPaperScissorsChoices[
+      Math.floor(Math.random()*rockPaperScissorsChoices.length)
+    ]
+
+    // Run functions to determine the winner
+    rockPaperScissors(playerChoice, botChoice)
+    bonusCheck()
+    updateMessage(playerChoice, botChoice, result)
+    roundWinner(playerScore, botScore)
+    updateScore()
+
+    // Log it
+    console.log(`Player's Choice: ${playerChoice}`)
+    console.log(`Bot's Choice: ${botChoice}`)
+    console.log(`Result of Game: ${resultMessage}`)
+    console.log(winLog);
+  })
+
+  // Functions and Things
+  function  rockPaperScissors(playerChoice, botChoice) {
+
+    // Rock, Paper, Scissor Game
+    if (playerChoice === botChoice) {
+      result = "tie"
+    } else if (playerChoice === "Rock") {
+        if (botChoice === "Paper") {
+          result = "bot"
+          botScore = botScore + 1
+        } else {
+          result = "player"
+          playerScore = playerScore + 1
+        }
+    } else if (playerChoice === "Paper") {
+      if (botChoice === "Scissors") {
+        result = "bot"
+        botScore = botScore + 1
+      } else {
+        result = "player"
+        playerScore = playerScore + 1
+      }
+    }  else if (playerChoice === "Scissors") {
+      if (botChoice === "Rock") {
+        result = "bot"
+        botScore = botScore + 1
+      } else {
+        result = "player"
+        playerScore = playerScore + 1
+      }
+    }
+
+    // Add Result to log
+    winLog.push(result)
+  }
+
+  function updateScore() {
```

Comment:
@jacobkelman great job using SRP (single responsibility principle) to keep the code modular and easy to follow